sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-spark_3-0_2-12</artifactId>
+  <version>4.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-spark_3-0_2-12;current} -->
+  <packaging>jar</packaging>
+  <name>OLTP Spark Connector for Azure Cosmos DB SQL API</name>
+  <description>OLTP Spark Connector for Azure Cosmos DB SQL API</description>
+  <url>https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cosmos/azure-cosmos-spark_3_2.12</url>
+
+  <properties>
+    <maven.build.timestamp.format>MM-dd-HH-mm-ss</maven.build.timestamp.format>
+    <jacoco.min.branchcoverage>0.17</jacoco.min.branchcoverage>
+    <jacoco.min.linecoverage>0.18</jacoco.min.linecoverage>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
We have Java 8 as baseline for all new Java SDKs and we also support JDK 11. Can we support both JDK 8 and 11 for this too?

Reply:
Spark 3.* only support Java 11 and newer. So the dependencies we need to pull in only support Java 11 and no customer would be able to use this runtime in a Spark environment not supporting Java < 11 anyway.